modules/virt-about-reclaiming-persistent-volumes.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/virtual_disks/virt-reclaiming-persistent-volumes.adoc
+
+[id="virt-about-reclaiming-persistent-volumes_{context}"]
+
+= About reclaiming persistent volumes
+
+When you reclaim a persistent volume (PV), you unbind the PV from a persistent volume claim (PVC)
+and delete the data in the network share. Finally, you bind the PV to a new PVC to
+prepare the volume for reuse.
+
+Kubernetes no longer recommends setting the Storage Class `reclaimPolicy`
+to `recycle` for persistent volume reclamation.
+Instead, use the following procedure to reclaim PVs.
```

Comment:
s/use the following procedure to/you can

Reply:
This is not an option. They must do it this way.